src/core/cstring.h

```diff
@@ -34,7 +34,7 @@ class CString
     explicit CString(const CString& cstr);
     ~CString();
 
-    uint8_t* data();
+    uint8_t* data() const;
```

Comment:
May be I don't understand something. But **this** method don't change object. This is a reason, why I make it as `const`